packages/rpc-transport-http/src/__tests__/http-transport-test.ts

```diff
@@ -71,6 +71,9 @@ describe('createHttpTransport', () => {
                 thrownError = e as SolanaError<typeof SOLANA_ERROR__RPC__TRANSPORT_HTTP_ERROR>;
             }
             expect(thrownError).toBeDefined();
+            
+            // Note: Stringifying this variable, it will leak header values and the test will fail
+            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions 
```

Comment:
> One of them I had to comment it because when it gets stringified, it leaks relevant information

That's the right thing to do though, because this is a test that tests exactly that, right?

Reply:
Ok, I guess it makes sense. I was a bit confused because a proper stringify would render the protected data.